### PR TITLE
SAK-29691 Fixing the crashing of assignment feed

### PR DESCRIPTION
**Pull request by pushyamig**

### CODE REVIEW

File: assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java [hunk 1]
```diff
@@ -295,32 +295,35 @@ public SimpleAssignment(Assignment a) {
...
-				}
-				else
-				{
-					// If the linked assignment is not internal to Gradebook, try the external assignment service
-					// However, there is no API available in GradebookExternalAssessmentService of getExternalAssignment()
-					// We will first check whether the external assignment is defined, and then get it through GradebookService 
-					boolean isExternalAssignmentDefined = gradebookExternalService.isExternalAssignmentDefined(a.getContext(), gradebookAssignmentProp);
-					if (isExternalAssignmentDefined)
+			if (gradebookService.isGradebookDefined(a.getContext())) {
+				if (gradebookAssignmentProp != null){
```
Comment (ottenhoff) [inline assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java]: Please use consistent braces style with existing code.

Comment (pushyamig) [inline assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/entityproviders/AssignmentEntityProvider.java]: Done

**Comment by lsloan** [comment]
Looks OK.  :shipit:

**Comment by ottenhoff** [comment]
Looks OK to me? Anyone else?

**Pull request merged**